tests/regression/roles/sonic_lldp_global/defaults/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+ansible_connection: httpapi
+module_name: lldp_global
+
+
+tests:
+  - name: test_case_01
+    description: Add Global LLDP configuration
+    state: merged
+    input:
+        enable: true
+        multiplier: 9
+        system_name: CR_sonic
+        hello_time: 18
+        mode: receive
+        system_description: Sonic_CR1_device
+        tlv_select: 
+            system_capabilities: false
+        
```

Comment:
Please remove trailing whitespaces in this file, cleanup_tests.yaml, preparation_tests.yaml and tasks_template_del.yaml